lib/kasket/visitor.rb

```diff
@@ -29,7 +29,7 @@ def visit_Arel_Nodes_SelectStatement(node, *_)
       return :unsupported if node.with
       return :unsupported if node.offset
       return :unsupported if node.lock
-      return :unsupported if node.orders.any?
+      return :unsupported if node.orders.reject { |o| o.is_a?(Arel::Nodes::Ascending) && o.expr.name == "id" }.any?
```

Comment:
This line is pretty hard to understand. How about extracting the block to a method – something like this?

```rb
node.orders.size > 1 && !default_ordering?(node.orders.first)
```